frontend/src/bundles/video_editor/components/timeline/subcomponents/timeaxis.tsx

```diff
@@ -0,0 +1,110 @@
+import { minutesToMilliseconds } from 'date-fns';
+import { useTimelineContext } from 'dnd-timeline';
+import { useMemo } from 'react';
+
+interface Marker {
+    label?: string;
+    sideDelta: number;
+    heightMultiplier: number;
+}
+
+interface MarkerDefinition {
+    value: number;
+    maxRangeSize?: number;
+    minRangeSize?: number;
+    getLabel?: (time: Date) => string;
+}
+
+interface TimeAxisProperties {
+    markers: MarkerDefinition[];
+}
+
+const TimeAxis = (properties: TimeAxisProperties): JSX.Element => {
+    const { range, direction, sidebarWidth, valueToPixels } = useTimelineContext();
+    const side = direction === 'rtl' ? 'right' : 'left';
+
+    const markers = useMemo(() => {
+        const sortedMarkers = [...properties.markers];
+        sortedMarkers.sort((a, b) => b.value - a.value);
```

Comment:
```suggestion
        const sortedMarkers = properties.markers.toSorted((a, b) => b.value - a.value);
```
In this way there will be no need to copy markers object because toSorted method does not mutate the array and return a new array.